src/Dialog/Dialog.js

```diff
@@ -46,6 +47,111 @@ export const styleSheet = createStyleSheet('MuiDialog', (theme) => {
   };
 });
 
+type MaxWidthBreakpoints = 'xs' | 'sm' | 'md';
+
+type DefaultProps = {
+  fullScreen: Breakpoint,
+  ignoreBackdropClick: boolean,
+  ignoreEscapeKeyUp: boolean,
+  enterTransitionDuration: number,
+  leaveTransitionDuration: number,
+  maxWidth: MaxWidthBreakpoints,
+  open: boolean,
+  transition: Function | Element<*>,
+};
+
+type Props = {
+  /**
+   * Dialog children, usually the included sub-components.
+   */
+  children?: Element<*>,
+  /**
+   * The CSS class name of the root element.
```

Comment:
What do you think about using `@ignore`? This is quite idiomatic and standard in the ecosystem. Why documenting it?